doc/releases/changelog-dev.md

```diff
@@ -151,6 +151,18 @@
   True
   ```
 
+* Added `qml.resource.estimate_error` and `qml.resource.estimate_samples` methods for
```

Comment:
Don't forget to add the link of the PR :)

Reply:
Oops! Added now.